autorag/evaluate/metric/generation.py

```diff
@@ -124,29 +125,46 @@ async def compute(gt: List[str], pred: str) -> float:
     return result
 
 
-@generation_metric
-def sem_score(generation_gt: List[str], pred: str, embedding_model: Optional[BaseEmbedding] = None) -> float:
+def sem_score(generation_gt: List[List[str]], generations: List[str],
+              embedding_model: Optional[BaseEmbedding] = None,
+              batch: int = os.cpu_count()) -> List[float]:
```

Comment:
I think default batch size is way more larger than cpu core count? 
Like 64 or 128? Because it uses CUDA, not cpu.